user_guide_src/source/outgoing/view_cells.rst

```diff
@@ -46,7 +46,7 @@ Simple Cells are classes that return a string from the chosen method. An example
     {
         public function show(array $params): string
         {
-            return "<div class="alert alert-{$params['type']}">{$params['message']}</div>";
+            return "<div class='alert alert-{$params['type']}'>{$params['message']}</div>";
```

Comment:
How about keep using double quote, escaped?

Reply:
That would work too.

Reply:
The use of single quotes is a bad practice and should not be used in documents.